eras/babbage/impl/src/Cardano/Ledger/Babbage/Tx.hs

```diff
@@ -63,11 +60,11 @@ instance AlonzoEraTx BabbageEra where
   isValidTxL = babbageTxL . isValidAlonzoTxL
   {-# INLINE isValidTxL #-}
 
-instance EraSegWits BabbageEra where
-  type TxSeq BabbageEra = AlonzoTxSeq BabbageEra
-  fromTxSeq = txSeqTxns
-  toTxSeq = AlonzoTxSeq
-  hashTxSeq = hashAlonzoTxSeq
+instance EraBlockBody BabbageEra where
```

Comment:
Could you please move this instance into its own `BlockBody` module for the sake of consistency with other eras